kbdd/src/test/kotlin/ru/fix/kbdd/rest/RestTest.kt

```diff
@@ -392,4 +393,31 @@ class RestTest {
                             """))
         )
     }
+
+    @Test
+    suspend fun `post multipart data`() {
+        request {
+            headers { "my-header" % "header-value" }
+            baseUri(server.baseUrl())
+            post("/post-form-data-request")
+            file("resource-with-placeholders.json", File("src/test/resources/resource-with-placeholders.json").inputStream())
+        }
```

Comment:
resource-with-placeholders.json is used by another test, it content can be changed or removed. 
Better to use new resource for new test. 
mulipart-data-file.json for example with some test content